test/bitmap-constructor-test.js

```diff
@@ -0,0 +1,52 @@
+'use strict';
+
+const expect = require('chai').expect;
+const index = require(`${__dirname}/../index.js`);
+const fileReader = require(`${__dirname}/../lib/bitmap-file-helper.js`);
+const bitmapConstructor = require(`${__dirname}/../model/bitmap-constructor.js`);
+
+describe('Bitmap Constructor Module', () => {
+  describe('#Invert Bitmap', () => {
+    it('Should have a new file written', (done) => {
+      fileReader.eraseFile(`${__dirname}/../assets/palette-invert-bitmap.bmp`);
+      index.invertBitmap();
+      fileReader.initFile(`${__dirname}/../assets/palette-invert-bitmap.bmp`, (err, data) => {
+        expect(err).to.equal(null);
```

Comment:
Instead of having all the invert bitmap do all the work and then re-reading the file to check if it is good you should structure your code a bit differently so that you can get the results of what you just wrote so you can easily test it without reading the file again. Right now you aren't proving that you aren't reading an empty string. 'I don't think you are but you want to show that you are.'